includes/discovery/os/mobileiron.inc.php

```diff
@@ -0,0 +1,10 @@
+<?php
+
+use Illuminate\Support\Str;
+
+// At ths time, MI don't make any customisations to the tree, so we just detect their packages
+if (Str::startsWith($device['sysDescr'], 'Linux') && Str::startsWith($device['sysObjectID'], '.1.3.6.1.4.1.8072.3.2.10')) {
```

Comment:
Not enthused about this detection, but at least you have explained it.

Reply:
Yeah, it's not ideal. I'm not sure how net-snmp decides to order the installed packages list, but it's not stable or obvious, so grabbing the whole branch of the tree is the only option.